tasks.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+from collections import namedtuple
+from datetime import date
+
+import requests
+import semver
+from invoke import call, task
+
+AppName = namedtuple("AppName", ["short", "long"])
+UNRELEASED_HEADER = "## [Unreleased]\n"
```

Comment:
Take or leave: we might want a constant for `## ` and call it something like `SECTION_MARKER`